torch/ao/quantization/fx/fusion_patterns.py

```diff
@@ -22,7 +23,7 @@ def __init__(self, quantizer: QuantizerCls, node: Node):
 
     @abstractmethod
     def fuse(self, quantizer: QuantizerCls, load_arg: Callable,
-             fuse_custom_config_dict: Dict[str, Any]) -> Node:
+             fuse_custom_config_dict: typing.Optional[Dict[str, Any]]=None) -> Node:
```

Comment:
we typically use `from typing import Optional`, can we change to that?

Reply:
Sure I'll change to that 💯

Reply:
It's done! 😄